apollo-client/v3/schema-driven-testing/src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { createRoot } from "react-dom/client";
+import { ApolloProvider } from "@apollo/client";
+import { client } from "./client";
+import { Products } from "./products";
+
+const rootElement = document.getElementById("root")!;
+const root = createRoot(rootElement);
+
+export const App = () => (
+  <ApolloProvider client={client}>
+    <main>
```

Comment:
It probably doesn't matter too much, but I noticed the other tests don't use `main` here. Should it stay consistent?

Reply:
Yes good call, don't need that here. Updated in https://github.com/apollographql/docs-examples/pull/128/commits/ebad17235a81d4407c691e9976f08f40f886f955.